src/functions/uploadToRSIS/application/result-client.ts

```diff
@@ -163,15 +164,21 @@ const mapHTTPErrorToDomainError = (err: AxiosError): Error => {
   return new Error(err.message);
 };
 
-const isRecordAutosaved = (test: StandardCarTestCATBSchema): BooleanAsNumber => {
-  // If the following properties are provided, we safely assume this is not an autosaved record.
-  // As testSummary maybe undefined, we use the non-null assertion operator.
+export const isRecordAutosaved = (test: StandardCarTestCATBSchema): BooleanAsNumber => {
+  const testDate = moment(new Date(test.journalData.testSlotAttributes.start));
+
+  // Two weeks or older is always flagged as autosave
+  if (moment().diff(testDate, 'days') >= 14) {
+    return 1;
+  }
+
+  // We assume that when these properties are null then the test is autosaved
   return (
-    get(test, 'testSummary.additionalInformation', false) &&
-    get(test, 'testSummary.candidateDescription', false) &&
-    get(test, 'testSummary.identification', false) &&
-    get(test, 'testSummary.independentDriving', false) &&
-    get(test, 'testSummary.routeNumber', false) &&
-    get(test, 'testSummary.weatherConditions', false))
-     ? 0 : 1;
+    (get(test, 'testSummary.additionalInformation', false) &&
```

Comment:
Is this not an extra parenthesis (at the beginning of this line) - there is on return level ?